config/action.d/badips.py

```diff
@@ -99,6 +104,7 @@ def __init__(self, jail, name, category, score=3, age="24h", key=None,
 		self.banaction = banaction
 		self.bancategory = bancategory or category
 		self.bankey = bankey
+		self.loglevel = str2LogLevel(loglevel) if isinstance(val, basestring) else loglevel
```

Comment:
Typo here with `val` instead of `loglevel` ?

Reply:
Thx, I saw it already by latest common review.